src/logger.js

```diff
@@ -251,11 +252,16 @@ class Logger extends Writable {
   log(lvl, log) {
     let modifiedLevel = lvl;
     let modifiedLog = log;
-    // Log is optional and not present, so we set the Log = Log Level
 
+    // If function is called without second argument, with just a message:
+    //  logger.log('oops');
+    //
+    //  Then we set the log level to null if we shouldn't take the level from the log,
+    //  since it doesn't exist.
+    //  We then place the message in the right variable.
     if (modifiedLog === undefined) {
       modifiedLog = modifiedLevel;
-      modifiedLevel = null;
+      modifiedLevel = this.takeLevelFromLog ? modifiedLog.level : null;
```

Comment:
Can modifiedLog we a string?

Reply:
It can, but I would say setting `takeLevelFromLog = true` and passing a string doesn't make sense.

It's also invalid based on example in README